packages/fhir-group-management/src/components/CommodityAddEdit/tests/__snapshots__/Form.test.tsx.snap

```diff
@@ -75,6 +75,56 @@ exports[`Health care form renders correctly: cancel button 1`] = `
 </button>
 `;
 
+exports[`Health care form renders correctly: id field 1`] = `
+<input
+  className="ant-input ant-input-disabled"
+  disabled={true}
+  id="id"
+  onBlur={[Function]}
+  onChange={[Function]}
+  onFocus={[Function]}
+  onKeyDown={[Function]}
+  placeholder="(Auto generated)"
+  type="text"
+  value=""
```

Comment:
Is this intentional?

Reply:
This is when creating a new commodity the value remains undefined and is only generated after clicking submit, so its never going to be in the form. However this attribute would have a value when editing a commodity